### PR TITLE
Export annotations to csv file in annotations_player plugin

### PR DESCRIPTION
**Pull request by snleu**

### CODE REVIEW

File: pupil_src/shared_modules/annotations.py [hunk 1]
```diff
@@ -188,6 +190,38 @@ def fire_annotation(self,annotation_label):
...
+
+
+    def export_annotations(self,export_range,export_dir):
+
+        if not self.annotations:
+            logger.warning('No annotations in this recording nothing to export')
+            return
+
+        annotations_in_section = chain(*self.annotations_by_frame[export_range])
+        annotations_in_section = dict([(a['index'],a) for a in annotations_in_section]).values() #remove dublicates
```
Comment (papr) [inline pupil_src/shared_modules/annotations.py]: One could use a dict-comprehension in here. That could be copied to the old code as well.

Comment (mkassner) [inline pupil_src/shared_modules/annotations.py]: @papr  can you post a line what what you have in mind. We are only removing dublicates and need the output as a list anyways.

Comment (papr) [inline pupil_src/shared_modules/annotations.py]: `annotations_in_section = {a['index']:a for a in annotations_in_section}.values()`

Comment (mkassner) [inline pupil_src/shared_modules/annotations.py]: nice!

**Comment by mkassner** [comment]
@snleu  once the comment by @papr  is implemented we can merge this.

**Pull request merged**